slides/writing-methods/index.html

```diff
@@ -0,0 +1,245 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+
+    <title>Writing methods - RM</title>
+
+    <link rel="stylesheet" href="../../css/reset.css">
+    <link rel="stylesheet" href="../../css/reveal.css">
+    <link rel="stylesheet" href="../../css/theme/rubymonstas.css">
+
+    <!-- Theme used for syntax highlighting of code -->
+    <link rel="stylesheet" href="../../lib/css/solarized.css">
+
+    <!-- Printing and PDF exports -->
+    <script>
+      var link = document.createElement( 'link' );
+      link.rel = 'stylesheet';
+      link.type = 'text/css';
+      link.href = window.location.search.match( /print-pdf/gi ) ? '../../css/print/pdf.css' : '../../css/print/paper.css';
+      document.getElementsByTagName( 'head' )[0].appendChild( link );
+    </script>
+  </head>
+  <body>
+    <div class="reveal">
+      <div class="slides">
+        <!-- Title slide with the topic. Don't forget to change the <title> tag too! -->
+        <section>
+          <div class="flex-container">
+            <div class="flex-column text-align-left">
+              <h1>Ruby Monstas</h1>
+              <hr>
+              <p class='subtitle'>
+                Writing methods
+              </p>
+            </div>
+            <div class="flex-column-200px">
+              <img class="no-border" src="../images/logo-cropped-small.png">
+            </div>
+          </div>
+          <aside class="notes">
+          </aside>
+        </section>
+
+        <section id="intro">
+          <h2>Intro</h2>
+          <p>We already learned how to use methods:</p>
+          <pre class="fragment fade-in"><code class="hljs ruby">"Popocatépetl".length                # => 12
+"Eyjafjallajökull".include?("kull")  # => true
+"Vesuvius".reverse                   # => "suivuseV"</code></pre>
+          <p class="fragment fade-in">Let's learn how to write our own, shall we?</p>
+        </section>
+
+        <section id="why-methods">
+          <h2>Why methods?</h2>
+          <p>Let's take a look at this example</p>
+          <pre class="fragment fade-in"><code class="hljs ruby">my_word = "Sky"
+puts "The word #{my_word} has #{my_word.length} letters"
```

Comment:
```suggestion
          <pre class="fragment fade-in"><code class="hljs ruby">word = "Sky"
puts "The word #{word} has #{word.length} letters"
```

we should change to `word`, right?